docs/docs/gestures/_shared/base-gesture-event-data.md

```diff
@@ -7,3 +7,11 @@ Current [state](/docs/fundamentals/states-events) of the handler. Expressed as o
 ### `numberOfPointers`
 
 Represents the number of pointers (fingers) currently placed on the screen.
+
+### `pointerType`
+Indicates the type of pointer device being used. This value is represented by one of the constants defined in the `PointerType` enum, which includes the following fields:
+
+- `TOUCH`
+- `STYLUS`
+- `MOUSE`
+- `OTHER`
```

Comment:
The first three are self-explanatory but I really would like to learn (and probably developers reading this documentation) what `OTHER` is.

Reply:
Done in [3a53822](https://github.com/software-mansion/react-native-gesture-handler/pull/2791/commits/3a538226389517f601c5658c85f9f542cd543607), let me know what you think!